com.unity.multiplayer.mlapi/Tests/Editor/BitStreamTests.cs

```diff
@@ -0,0 +1,674 @@
+using System;
+using NUnit.Framework;
+using MLAPI.Serialization;
+using System.Text;
+
+public class BitStreamTests
+{
+    [Test]
+    public void TestEmptyStream()
+    {
+        BitStream bitStream = new BitStream(new byte[100]);
+        // ideally an empty stream should take no space
+        Assert.That(bitStream.Length, Is.EqualTo(100));
```

Comment:
When I read this comment, I then expect the next line should expect bitStream.Length to be 0 not 100